### ISSUE TITLE
Tutorial Links not working

### ISSUE THREAD

**Issue by Uveso** [FAForever/client#394]
I hope this is the right place for posting this :)

Issue from 02.Mar (4 month ago):
http://forums.faforever.com/viewtopic.php?f=45&t=11860

Inside FA Forever, if i klick on Tutorials, there are 2 links. Both are not working:

```
http://www.faforever.com/forums/viewtopic.php?f=2&t=537
http://www.faforever.com/forums/viewtop ... f=2&t=1614
```

Should be changed to:

```
http://forums.faforever.com/viewtopic.php?f=2&t=537
http://forums.faforever.com/viewtopic.php?f=2&t=1614

```

**Comment by Crotalus** [comment]
Yup right place but already fixed here 3cfbe9d58b20c8f5913f49aadb9b2e066f7952e0 in `develop` so next official client update will solve this.

**Comment by Uveso** [comment]
Perfect!
I was cleaning my ToDo list, and this was one of the last things on there :D
